src/cli/commands/pin/rm.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+module.exports = {
+  command: 'rm <ipfs-path>',
+
+  describe: 'Removes the pinned object from local storage.',
+
+  builder: {
+    recursive: {
+      type: 'boolean',
+      alias: 'r',
+      default: true,
+      describe: 'Recursively unpin the objects linked to by the specified object(s).'
+    }
+  },
+
+  handler: (argv) => {
+    const paths = argv['ipfs-path'].split(' ')
+    const recursive = argv.recursive
+    argv.ipfs.pin.rm(paths, { recursive }, (err, results) => {
+      if (err) { throw err }
```

Comment:
Same as above. Let's make sure that these error messages are printed in a way that it provides useful information to the user and it doesn't look like the whole program crashed

Reply:
Since it looks like all the CLI modules just throw the error, would that be better as a separate PR to replace all of them with some consistent response?

Reply:
Sounds good 👍